src/patternfly/components/Form/docs/code.md

```diff
@@ -21,6 +21,6 @@
 | `.pf-c-form__helper-text` | `<p>` |  Initiates a form helper text block. |
 | `.pf-c-form__section` | `<section>`, `<div>` |  Initiates a form section. For larger forms, section grouping may be necessary. This class wraps chunks of code and separates them with padding. |
 | `.pf-c-form__group` | `<div>` |  Wraps form fields `<label>` + `<field>` + `.helper-text` __(not checkboxes or radios)__. Form groups should have no more than one of each element type. Form groups can be nested and, for more complex fieldsets, it may be necessary to do so. |
-| `.pf-m-error` | `.pf-c-form__helper-text`, `.pf-c-form__input` | Modifies form element border and background-image. |
+| `.pf-m-error` | `.pf-c-form__helper-text`| Modifies text color of helper text. |
```

Comment:
I can see the following is defined in the .scss file, but I can't seem to find where that color variable is getting defined. Is this text meant to be red? It doesn't display as red.
```
     &.pf-m-error {	    
       color: var(--pf-c-form--m-error--Color);
     }
```

Reply:
ah thanks! ya I missed that one, the variables changed since I worked on this last :(